### PR TITLE
LPS-77931 Editable images

### PR DESCRIPTION
**Pull request by p2kmgcl**
/cc @ealonso

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test

File: modules/apps/web-experience/layout/layout-admin-web/src/main/resources/META-INF/resources/js/fragments_editor/FragmentEntryLink.es.js [hunk 1]
```diff
@@ -60,9 +71,20 @@ class FragmentEntryLink extends Component {
...
+			AUI().use(
+				'aui-parse-content',
+				A => {
+					const content = A.one(this.refs.content);
+					content.plug(A.Plugin.ParseContent);
+					content.setContent(this.content);
+
+					this._processors.forEach(
+						processor => {
+							processor.process();
```
Comment (jbalsas) [inline modules/apps/web-experience/layout/layout-admin-web/src/main/resources/META-INF/resources/js/fragments_editor/FragmentEntryLink.es.js]: This feels a bit weird... what are you processing? Why not change this API so the `FragmentProcessor` doesn't need the fragmentEntry on the constructor and simply accepts it on for `process(fragmentEntryLink)`?

File: modules/apps/web-experience/layout/layout-admin-web/src/main/resources/META-INF/resources/js/fragments_editor/fragment_processors/FragmentProcessor.es.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/**
+ * Class that can perform some changes over a fragmentEntryLink
+ * while it is being editable.
+ * @review
+ */
+
+class FragmentProcessor {
```
Comment (jbalsas) [inline modules/apps/web-experience/layout/layout-admin-web/src/main/resources/META-INF/resources/js/fragments_editor/fragment_processors/FragmentProcessor.es.js]: Do we really need this abstract class? If so, we should at least mark it as `@abstract` so it's clear you're supposed to always extend it. It actually pretty much will work as an interface if we move the `constructor` logic to the `process` API as per the comment above.

Why do we need an extra `create` lifecycle? Isn't the constructor enough?

Comment (p2kmgcl) [reply modules/apps/web-experience/layout/layout-admin-web/src/main/resources/META-INF/resources/js/fragments_editor/fragment_processors/FragmentProcessor.es.js]: I will remove this abstract class and keep just the different processors :+1:

File: modules/apps/web-experience/layout/layout-admin-web/src/main/resources/META-INF/resources/js/fragments_editor/FragmentEntryLink.es.js (hunk 1 above)
Comment (p2kmgcl) [reply modules/apps/web-experience/layout/layout-admin-web/src/main/resources/META-INF/resources/js/fragments_editor/FragmentEntryLink.es.js]: I will make those processors static, and make them receive the fragment instance inside the process method :+1:

**Pull request closed without merging**